.travis.yml

```diff
@@ -0,0 +1,48 @@
+sudo: required
+language: python
+jdk: openjdk8
+services: docker
+python: '3.6'
+cache: pip
+matrix:
+  fast_finish: true
+
+before_install:
+  # PRs made to 'master' branch should always orginate from another repo or the 'dev' branch
+  - '[ $TRAVIS_PULL_REQUEST = "false" ] || [ $TRAVIS_BRANCH != "master" ] || ([ $TRAVIS_PULL_REQUEST_SLUG = $TRAVIS_REPO_SLUG ] && ([ $TRAVIS_PULL_REQUEST_BRANCH = "dev" ] || [ $TRAVIS_PULL_REQUEST_BRANCH = "patch" ]))'
+  # Pull the docker image first so the test doesn't wait for this
+  - docker pull nfcore/neutronstar:dev
+  - docker pull nfcore/supernova:latest
```

Comment:
Would be nice to tag this docker image, just in case there is a new `:latest` in the future..

Reply:
I've tagged it according to the supernova version of the container